src/main/java/seedu/address/model/date/BirthDate.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.date;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class BirthDate extends DocumentedDate {
+    private static final DateTimeFormatter FORMATTER_INPUT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    public final String value;
+    public final String jsonValue;
```

Comment:
Does this `jsonValue` lower the cohesiveness of BirthDate? Like a model type doing storage task? Maybe give a general name. My advice is to use just use `value`, format it in the `FORMATTER_INPUT` format. Use `toString` for display.